src/IO/ImageDataFromChunks.worker.js

```diff
@@ -46,74 +42,75 @@ registerWebworker().operation(
       pixelArray = new pixelArrayType(pixelArrayElements)
     }
 
+    const arrayShape = Object.fromEntries(info.arrayShape)
+    const pixelStrides = {
+      z: arrayShape.c * arrayShape.x * arrayShape.y,
+      y: arrayShape.c * arrayShape.x,
+      x: arrayShape.c,
+    }
+
+    const chunkSize = Object.fromEntries(info.chunkSize)
+    const chunkStrides = {
```

Comment:
The chunk strides for each dimension will be based on the input MultiscaleSpatialImage `dims` and their order: maybe a `slice` here on the `dims` array starting on the `dim` index and a `reduce` based on the `chunkSize`.